### DISCUSSION TITLE
How many Agents does a Collector support at most?

### DISCUSSION THREAD

**Discussion by wanglei13975** [naver/scavenger#119] [Q&A] [answered]
Have you conducted performance DFX tests for a large number of Agents? 
What is the maximum number of Agents that a Collector can support?
Have you considered any plans to change the communication between Agent and Collector to a message queue?

**Comment by taeyeon-Kim** [comment, marked as answer]
@wanglei13975 

- We haven't done any performance testing between agents and collectors, but we have 3 collectors handling requests from roughly 4400 agents quite comfortably on our own.

- In theory, one collector is equivalent to a Tomcat handling request.

- We haven't made any plans yet, but thanks for the idea.

**Reply by wanglei13975** [reply]
Thank you for your reply.
